seed/seed.json

```diff
@@ -390,7 +390,7 @@
             "experiments": [
                 {
                     "name": "DefaultAdNotificationsPerHour=10/MaximumAdNotificationsPerDay=100/MaximumInlineContentAdsPerHour=12/MaximumInlineContentAdsPerDay=60/AdServingVersion=1",
-                    "probability_weight": 90,
+                    "probability_weight": 50,
```

Comment:
you need to change other experiment in this study so overall weight would become 100.

Reply:
Great catch, thanks. Fixed